docs/modelserving/v1beta1/llm/vllm/README.md

```diff
@@ -0,0 +1,81 @@
+## Deploy the LLaMA model with vLLM Runtime
+Serving LLM models can be surprisingly slow even on high end GPUs, [vLLM](https://github.com/vllm-project/vllm) is a fast and easy-to-use LLM inference engine. It can achieve 10x-20x higher throughput than Huggingface transformers.
+It supports [continuous batching](https://www.anyscale.com/blog/continuous-batching-llm-inference) for increased throughput and GPU utilization,
+[paged attention](https://vllm.ai) to address the memory bottleneck where in the autoregressive decoding process all the attention key value tensors(KV Cache) are kept in the GPU memory to generate next tokens.
+
+You can deploy the LLaMA model with built vLLM inference server container image using the `InferenceService` yaml API spec. 
+We have work in progress integrating `vLLM` with `Open Inference Protocol` and KServe observability stack.
+
+The LLaMA model can be downloaded from [huggingface](https://huggingface.co/meta-llama/Llama-2-7b) and upload to your cloud storage.
+
+=== "Yaml"
```

Comment:
YAML is not correctly rendering.

<img width="1084" alt="Screenshot 2023-10-24 at 10 10 46 AM" src="https://github.com/kserve/website/assets/8099140/3e45bcf5-1fd0-4950-892d-886a1da5c144">